orttraining/orttraining/training_ops/cuda/aten_ops/aten_op.cc

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "orttraining/training_ops/cpu/aten_ops/aten_op.h"
+#include "core/providers/cuda/cuda_fwd.h"
+
+namespace onnxruntime {
+namespace cuda {
+
+ONNX_OPERATOR_KERNEL_EX(
+    ATenOp, kMSDomain, 1, kCudaExecutionProvider,
+    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::AllTensorAndSequenceTensorTypes()).ExternalOutputs(),
```

Comment:
I wonder do we need to set ExternalOutputs() here ?

As we are using aten kernels as a function, pytorch shouldn't need keep external reference of the output tensor?

Reply:
We uses SetOutputMLValue to set the OrtValue back, and SetOutputMLValue only allows setting external outputs. I am thinking external output means the output is from external (not allocated by ORT), whether the creator still hold the reference of the tensor is not important.